### PR TITLE
fix: [CDS-37140]: Displayed store info in Yaml view of triggers under…

### PR DESCRIPTION
**Pull request by goyalsonali**
… source

##### Summary:

Earlier, store info was missing inside source due to which inavlid yaml warning was shown after saving trigger.
Now, fetched the value of store from resolved pipeline to show it under source.

##### Jira Links:

https://harness.atlassian.net/browse/CDS-37140

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by goyalsonali** [comment]
retrigger titlecheck

File: src/modules/72-triggers/pages/triggers/TriggersWizardPage.tsx [hunk 1]
```diff
@@ -261,16 +261,72 @@ const getArtifactManifestTriggerYaml = ({
...
   })
 
-  // clears any runtime inputs
-  const artifactSourceSpec = clearRuntimeInputValue(
+  const filteredStagesforStore = val?.resolvedPipeline?.stages
+
+  //if manifest chosen is from stage
+  const filteredManifestforStore = filteredStagesforStore?.map(
+    (st: { stage: { spec: { serviceConfig: { serviceDefinition: { spec: { manifests: any[] } } } } } }) =>
+      st?.stage?.spec?.serviceConfig?.serviceDefinition?.spec?.manifests?.find(
```
Comment (mtran7) [inline src/modules/72-triggers/pages/triggers/TriggersWizardPage.tsx:269]: We can also use get from lodash

get(st, 'stage.spec.serviceConfig.serviceDefinition.spec.manifests' ||[]).find(...)

File: src/modules/72-triggers/pages/triggers/TriggersWizardPage.tsx [hunk 2]
```diff
@@ -261,16 +261,72 @@ const getArtifactManifestTriggerYaml = ({
...
+
+  const storeManifest = filteredManifestforStore?.find((mani: undefined) => mani != undefined)
+  let storeVal = storeManifest?.manifest?.spec?.store
+
+  //if manifest chosen is of parallel stage then to show store value in trigger yaml
+  const filteredParallelManifestforStore = filteredStagesforStore?.map((st: { parallel: any[] }) =>
+    st?.parallel
+      // eslint-disable-next-line @typescript-eslint/no-shadow
+      ?.map(st =>
+        st?.stage?.spec?.serviceConfig?.serviceDefinition?.spec?.manifests?.find(
```
Comment (mtran7) [inline src/modules/72-triggers/pages/triggers/TriggersWizardPage.tsx:282]: same comment as above

File: src/modules/72-triggers/pages/triggers/TriggersWizardPage.tsx [hunk 3]
```diff
@@ -261,16 +261,72 @@ const getArtifactManifestTriggerYaml = ({
...
+          spec: { ...selectedArtifact?.spec, store: storeVal }
         }) || ''
       )
     )
   )
+
+  //if connectorRef present in store is runtime then we need to fetch values from stringifyPipelineRuntimeInput
+  const filteredStageforRuntimeStore = parse(stringifyPipelineRuntimeInput)?.pipeline?.stages?.map(
+    (st: { stage: { spec: { serviceConfig: { serviceDefinition: { spec: { manifests: any[] } } } } } }) =>
+      st?.stage?.spec?.serviceConfig?.serviceDefinition?.spec?.manifests?.find(
```
Comment (mtran7) [inline src/modules/72-triggers/pages/triggers/TriggersWizardPage.tsx:314]: same comment as above

File: src/modules/75-cd/components/PipelineSteps/K8sServiceSpec/ManifestSource/ManifestSourceUtils.ts [hunk 4]
```diff
@@ -69,12 +69,28 @@ export const getManifestTriggerSetValues = (
     const initialArtifactValue = get(initialValues, `${manifestPath}`)
     const { selectedArtifact } = formik?.values || {}
 
-    if (initialArtifactValue && selectedArtifact.identifier === initialArtifactValue.identifier) {
+    const stages = formik?.initialValues?.inputSetTemplateYamlObj?.pipeline?.stages
+    const stageConnectorManifest = stages?.map(
+      (st: { stage: { spec: { serviceConfig: { serviceDefinition: { spec: { manifests: any[] } } } } } }) =>
+        st?.stage?.spec?.serviceConfig?.serviceDefinition?.spec?.manifests?.find(
```
Comment (mtran7) [inline src/modules/75-cd/components/PipelineSteps/K8sServiceSpec/ManifestSource/ManifestSourceUtils.ts:75]: same comment as above

**Review by mtran7 — APPROVED** [review]

**Review by mtran7 — APPROVED** [review]

**Comment by goyalsonali** [comment]
trigger cypress

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**